lib/cli.coffee

```diff
@@ -25,12 +26,20 @@ installMycha = (testHelperPath, done) ->
   installer.install done
 
 
+loadConfig = (done) ->
```

Comment:
There is already a ConfigLoader class. Why not make this method a class method on that class, and get rid of this method here? It doesn't really do anything except `new ConfigLoader(cwd: process.cwd()).load done` anyways.